cmd/nri-ecs.go

```diff
@@ -6,17 +6,35 @@ import (
 	"strings"
 	"time"
 
-	v3 "source.datanerd.us/fsi/nri-ecs/internal/metadata/v3"
-	"source.datanerd.us/fsi/nri-ecs/pkg/ecs"
+	sdkArgs "github.com/newrelic/infra-integrations-sdk/args"
+	"github.com/newrelic/infra-integrations-sdk/integration"
+	"github.com/newrelic/infra-integrations-sdk/log"
+
+	v3 "github.com/newrelic/nri-ecs/internal/metadata/v3"
+	"github.com/newrelic/nri-ecs/pkg/ecs"
+)
+
+var (
+	integrationName    = "com.newrelic.ecs"
+	integrationVersion = "1.3.1"
```

Comment:
can we inject it with build flags?

Reply:
yes! i didn't work on it yet to avoid to many changes in the same PR and incrementally working on this

Reply:
btw this is not a new version , this is actually the current release version

Reply:
your call, for that reason is approved already 😄